gap/PackageManager.gi

```diff
@@ -1057,7 +1057,10 @@ function()
   if PKGMAN_CustomPackageDir <> "" then
     dir := PKGMAN_CustomPackageDir;
   else
-    dir := UserHomeExpand("~/.gap/pkg");  # TODO: cygwin?
+    if GAPInfo.UserGapRoot = fail then
+      ErrorNoReturn("UserGapRoot not set. Cannot determine package directory");
```

Comment:
Why not fall back to what we did before?

Reply:
I think it UserGapRoot should always be set -- and if it isn't, then GAP probably isn't going to load packages from it (which is what was happening in cygwin -- packages appeared to install fine into ~/.gap/pkg, but then couldn't be loaded as that wasn't a place gap loaded from).

However, there could be some OS / situation which requires the fallback, so I'll put it back just to be safe.